### PR TITLE
 Styling and rendering the image for each article 

### PR DESCRIPTION
**Pull request by Cma-B**
### User Story
```
As a user 
In order to have better UI experience
I would like to see the image for each article and a bit of styling 
```
## Changes proposed in this pull request:
* styling and rendering image
Screen Shot 2020-08-23 at 11 51 46 AM

### CODE REVIEW

**Review by Felix-Willborg — APPROVED** [review]

File: cypress/integration/visitorCanSeeListOfArticles.feature.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ describe("Vistors can see list of Articles", () => {
       cy.get("#title").should("contain", "Scrum Lord");
       cy.get("#lead").should("contain", "Lord of all coharts");
       cy.get("button").should("contain", "Read more").click();
+      
     });
```
Comment (viktorsommar) [inline cypress/integration/visitorCanSeeListOfArticles.feature.js]: extra space

File: cypress/integration/visitorCanSeeListOfArticles.feature.js [hunk 2]
```diff
@@ -22,6 +23,7 @@ describe("Vistors can see list of Articles", () => {
       cy.get("#lead").should("contain", "Happy campers is always a winner");
       cy.get("button").should("contain", "Read more").click();
     });
+
```
Comment (viktorsommar) [inline cypress/integration/visitorCanSeeListOfArticles.feature.js]: extra space

**Review by Alex-Bellamy — CHANGES_REQUESTED** [review]
perhaps some screenshots and formatting

File: src/components/ArticleContent.jsx [hunk 3]
```diff
@@ -14,6 +15,7 @@ const ArticleContent = (props) => {
     if (isUserAuthenticated === false) {
       articleContent = (
         <>
+
```
Comment (viktorsommar) [inline src/components/ArticleContent.jsx]: extra space

File: src/components/ArticleContent.jsx [hunk 4]
```diff
@@ -33,24 +35,33 @@ const ArticleContent = (props) => {
...
 
   return (
-    <div className="article-list">
-      <div id={`article-${props.article.id}`} data-id={props.article.id}>
+   <Segment>
+    <Grid className="article-list">
+      <Grid.Column width={4}>
+      <Image src={props.article.image}/> 
+      </Grid.Column>
+
```
Comment (viktorsommar) [inline src/components/ArticleContent.jsx:46]: extra space

File: src/components/ArticleContent.jsx [hunk 5]
```diff
@@ -33,24 +35,33 @@ const ArticleContent = (props) => {
...
+      <Grid.Column width={9} id={`article-${props.article.id}`} data-id={props.article.id}>
         <h1 id="title">{props.article.title}</h1>
+        <Divider horizental></Divider>
         <h2 id="lead">{props.article.lead}</h2>
+
```
Comment (viktorsommar) [inline src/components/ArticleContent.jsx]: extra space

File: src/components/ArticleContent.jsx [hunk 6]
```diff
@@ -33,24 +35,33 @@ const ArticleContent = (props) => {
...
-            <button id="button" onClick={props.getSingleArticle}>
+            <Button id="button" onClick={props.getSingleArticle}>
               Read more
-            </button>
+            </Button>
           )}
-      </div>
-    </div>
+      </Grid.Column>
+
```
Comment (viktorsommar) [inline src/components/ArticleContent.jsx]: extra space

**Review by viktorsommar — CHANGES_REQUESTED** [review]

**Comment by Cma-B** [comment]
> perhaps some screenshots and formatting

both done!

**Pull request merged**